dev/config/keycloak/import/ltt-realm.json

```diff
@@ -51,6 +51,19 @@
     "clientRole": false,
     "containerId": "25facb13-fdb5-46cd-b66d-fa9305652f1a"
   },
+  "roles": {
```

Comment:
Why not create `patient-realm-role` too?

Reply:
I'm hesitant to add a patient role unless it fulfills a requirement. Roles can have different meanings to different projects/services and I want to make sure we don't paint ourselves into a corner with assumptions about LTT.

Reply:
dhair2 does want to see that role now, but I think you know that. I assumed you intended this to be a patient user, perhaps I was wrong there. No worries, probably easier to clarify at our next call.